runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -51,12 +52,25 @@
 @Slf4j
 @PluginDescriptor(
 	name = "Interface Styles",
-	description = "Change the interface style to the 2005/2010 interface",
-	tags = {"2005", "2010", "skin", "theme", "ui"},
+	description = "Change the interface style to the 2005/2010 interface and toggle member skills",
```

Comment:
This could use further explanation. "toggle member skills" doesn't mean a lot on its own.